script.js

```diff
@@ -0,0 +1,76 @@
+function startGame() {
+
+  for(var i = 1; i <= 9; i = i + 1) {
+    clearBox(i);
+  }
+
+  document.turn = "X";
+  if (Math.random()< 0.5) {
+    document.turn = "O";
+  }
+  document.winner = null;
+  setMessage(document.turn + " gets to start.");
+
+}
+
+function setMessage(msg) {
+  document.getElementById("message").innerHTML = msg;
+}
+
+function nextMove(square) {
+  if (document.winner !== null) {
+    setMessage(document.winner + " already won the game.");
+  } else if (square.innerHTML == "") {
+    square.innerHTML = document.turn;
+    switchTurn();
+  } else {
+    setMessage("That square is not empty.");
+  }
+}
+
+function switchTurn() {
+  if (checkWinner(document.turn)) {
+    setMessage("Congratulations, " + document.turn + "! You win!");
+    document.winner = document.turn;
+  } else if (document.turn == "X") {
+    document.turn = "O";
+    setMessage("It's " + document.turn + "'s turn.")
+  }
+  else {
+    document.turn = "X";
+    setMessage("It's " + document.turn + "'s turn.")
+  }
+
+}
+
+function checkWinner(move) {
+  var result = false;
+  if (checkRow(1, 2, 3, move) ||
+      checkRow(4, 5, 6, move) ||
+      checkRow(7, 8, 9, move) ||
+      checkRow(1, 4, 7, move) ||
+      checkRow(2, 5, 8, move) ||
+      checkRow(3, 6, 9, move) ||
+      checkRow(1, 5, 9, move) ||
+      checkRow(3, 5, 7, move)) {
```

Comment:
This duplication is asking for another abstraction.  What do you think about `winningCombinations` which returns an array of the possibilities?